includes/Modules/Idea_Hub.php

```diff
@@ -212,6 +212,31 @@ public function register() {
 			 */
 			add_filter( 'post_class', $this->get_method_proxy( 'update_post_classes' ), 10, 3 );
 
+			add_filter(
+				'googlesitekit_inline_base_data',
+				function( $data ) {
+					if (
+						// Do nothing if tracking is disabled or if it is enabled and already allowed.
+						empty( $data['trackingEnabled'] ) ||
+						! empty( $data['trackingAllowed'] ) ||
+						// Also do nothing if the get_current_screen function is not available.
+						! function_exists( 'get_current_screen' )
+					) {
+						return $data;
+					}
+
+					$screen = get_current_screen();
+					if ( ! is_null( $screen ) ) {
+						$data['trackingAllowed'] =
+							( 'post' === $screen->post_type && 'edit-post' === $screen->id ) ||
+							'dashboard' === $screen->id;
```

Comment:
This is still obviously kind of hacky overriding only the JS value, but it makes sense given we don't want to introduce a dedicated filter at this point. Let's open a follow-up issue revisiting how we should treat core screens that have some very limited Site Kit usage. It's not necessarily only about this, but also about loading certain assets on core screens, where I'm sure we can improve the situation by building a proper API.

cc @aaemnnosttv